src/native/libs/System.Native/entrypoints.c

```diff
@@ -215,8 +215,8 @@ static const Entry s_sysNative[] =
     DllImportEntry(SystemNative_SchedSetAffinity)
     DllImportEntry(SystemNative_SchedGetAffinity)
     DllImportEntry(SystemNative_GetProcessPath)
-    DllImportEntry(SystemNative_GetNonCryptographicallySecureRandomBytes)
-    DllImportEntry(SystemNative_GetCryptographicallySecureRandomBytes)
+    DllImportEntry(minipal_get_non_cryptographically_secure_random_bytes)
+    DllImportEntry(minipal_get_cryptographically_secure_random_bytes)
```

Comment:
Please keep the actual entry points `SystemNative_*` and have their implementations call `minipal_*`.